### PR TITLE
Added skills section

### LINKED ISSUE #19: Add Skills Section
We need to add skills section after education and before projects. 
Take help of this template.
https://youssefraafatnasry.me/portfolYOU/about/

### PR DESCRIPTION
**Pull request by VasaviLagishetty**
skills
mobile_view_skills
resume

### CODE REVIEW

File: _includes/about.html [hunk 1]
```diff
@@ -5,6 +5,8 @@
   <div class="row">
     <div class="col-md-10 col-md-offset-1">
       {{ page.content | markdownify }}
+
+      <a class="project-link" id="resume" href="{{ site.resume }}" target="_blank">Resume <i class="fa fa-download fa-fw"></i></a>
```
Comment (SriSatyaLokesh) [inline _includes/about.html]: we need to improve the CSS for download resume button

Comment (SriSatyaLokesh) [reply _includes/about.html]: this is for in future. for now it's fine

**Review by puneethkanna — COMMENTED** [review]
Nice!, but if only some logos are used, the recruiter gets confused. We need to use all of them or we will just use the filled bar to show them

File: about.html [hunk 2]
```diff
@@ -7,12 +7,5 @@
 
 Here at DevsKrate we inspire and endeavor to share everything about technology like Softwares, Coding, etc., and provide our best about HOW TO tutorials and BEST OF every category in accordance to our knowledge.
 
-<small>
-Octocat is Github's logo and trademark and here is used to illustrate a fictional character (John Smith) that used the *{ Personal }* Jekyll Theme to build his website.
-</small>
```
Comment (SriSatyaLokesh) [inline about.html:12]: why dont keep this small tags, maybe we wil forget this in future

**Review by SriSatyaLokesh — APPROVED** [review]

**Comment by SriSatyaLokesh** [comment]
> Nice!, but if only some logos are used, the recruiter gets confused. We need to use all of them or we will just use the filled bar to show them

we tried this, we will keep logos according to users skills

File: about.html (hunk 2 above)
Comment (VasaviLagishetty) [reply about.html:12]: will add again

**Review by puneethkanna — APPROVED** [review]
I think it need to be changed before merging

**Comment by SriSatyaLokesh** [comment]
> I think it need to be changed before merging

Do you mean by bar for each skill?

**Comment by puneethkanna** [comment]
> > I think it need to be changed before merging
> 
> Do you mean by bar for each skill?

I have typed before seeing the above message. It is by mistake

**Pull request merged**